hw_twitterapi/twitterapiapp/templates/twitterapiapp/main_page.html

```diff
@@ -64,14 +64,16 @@ <h1 class="mt-1">Recent Favorite Statistics of a User<span class="text-muted">(b
         <p>API call: <code><a href="http://206.189.12.235/api/recent_favorites/screen_name?count=20" id="recent_favorites_api">http://206.189.12.235/api/recent_favorites/screen_name?count=20</a></code></p>
         <p>Recent Favorite Statistics of a User.</p>
 
-        {% comment "Hide unmerged api documentation." %}
         <hr>
 
-        <h1 class="mt-1">Followers of a User</h1>
-        <p>Web interface: <code><a href="http://206.189.12.235/followers/username?count=10">http://206.189.12.235/followers/username?count=10</a></code></p>
-        <p>API call: <code><a href="http://206.189.12.235/api/followers/username">http://206.189.12.235/api/followers/username</a></code></p>
+        <<h1 class="mt-1">Followers of a User<span class="text-muted">(by <a href="https://github.com/isikz">isikz</a>)</span></h1>
+        <div class="form-inline my-2">
+            <input class="form-control" type="text" placeholder="Username" id="followers=username">
+            <button class="btn btn-primary ml-2" data-url="" id="followers_send">Send</button>
+        </div>
+        <p>Web interface: <code><a href="http://206.189.12.235/find_followers/follower?count=20" id="followers_web">http://206.189.12.235/followers/screen_name?count=20</a></code></p>
```

Comment:
you have `?count=20` in your url pattern, you haven't provided an input box for that purpose